### PR TITLE
update post methods, trim httputils

### PR DESCRIPTION
**Pull request by okjintao**
get, post requests should be really easy now. the last thing to do is to replace the `Model` objects in the current `<T>Response` buckets being used for the different controllers with the new `Response` objects and return a real object from the dto.

### CODE REVIEW

File: src/main/java/io/ark/core/model/Account.java [hunk 1]
```diff
@@ -1,68 +1,49 @@
...
 	private String publicKey;
-	private int unconfirmedSignature;
-	private int secondSignature;
-	private String secondPublicKey;
-	private List<String> multisignatures;
-	private List<String> u_multisignatures;
-	
-	private NetworkInfo info;
-	private ECKey keyPair;
+  private int unconfirmedSignature;
```
Comment (okjintao) [inline src/main/java/io/ark/core/model/Account.java]: Sorry for this massacre. I'm going to add some editor configs probably in a contribution area along with checkstyle rules so this nastiness doesn't happen again. (i'll fix this in the next pr no need for a tiny commit for this)

**Comment by okjintao** [comment]
after those style changes the pr is quite large but mostly just indentation changes etc. going to branch off here and work on peer selection methods as well as read up on v2 serialization. will open another pr with those changes.

File: src/main/java/io/ark/core/model/Transaction.java [hunk 2]
```diff
@@ -30,20 +28,20 @@
...
     this.fee = TX_FEE;
     this.recipientId = recipientId;
     this.timestamp = getTime();
     this.senderPublicKey = keyPair.getPublicKeyAsHex();
   }
 
   public TransactionDTO convert() {
-    return new TransactionDTO(id, type, timestamp, amount, fee, recipientId, senderPublicKey, signature, signSignature,
-        new Asset());
+    return new TransactionDTO(id, type, timestamp, amount, fee, recipientId, senderPublicKey,
```
Comment (dafty) [inline src/main/java/io/ark/core/model/Transaction.java]: Can we get rid of the 80 char line wrapping rule. I feel like it's a legacy convention and just makes code nasty to look at while no longer serving any functional purpose.

File: src/main/java/io/ark/core/requests/AccountManager.java [hunk 3]
```diff
@@ -1,108 +1,113 @@
...
   }
 
-  public List<String> createPassphrase() {
-    return Crypto.getPassphrase();
-  }
-  
-  public Account fetchAccount(String secret) {
-    return createAccount(secret);
+  public String createPassphrase() {
+    return String.join(" ", Crypto.getPassphrase());
```
Comment (dafty) [inline src/main/java/io/ark/core/requests/AccountManager.java]: I don't think this does anything.

**Review by dafty — CHANGES_REQUESTED** [review]
We need to consider our types for amounts (balances, values etc) and try to make them generic. This will make it more consistent to work with and prevent things like integer overflows when sending / retrieving large amounts.

**Comment by dafty** [comment]
We need to consider our types for amounts (balances, values etc) and try to make them generic. This will make it more consistent to work with and prevent things like integer overflows when sending / retrieving large amounts (for example, the current supply is 13196198600000000). I've used a Long for all monetary values in the Jackson objects.

**Comment by okjintao** [comment]
@dafty do you want to then keep all units as `long` then since that's what they come back as? i was doing the conversions to the GArk(?) for them on some methods and not others so yeah i feel the concern.

**Comment by okjintao** [comment]
@dafty 

- Updated wrap length to 240 (basically removed the constraint since 240 is huge IMO)
- Remove all conversions from managers (long in response is used)

**Comment by dafty** [comment]
@cdvallejo thanks!

**Review by dafty — APPROVED** [review]

**Comment by dafty** [comment]
I think long is a good replacement for now since all amounts returned are natural numbers (no decimals).

In the future, we could overload utility functions such as `createTransaction` with an `Denomination.ARKTOSHI` or `Denomination.ARK` enum which accepts other datatypes (like a double) and do the conversion within the function itself:

`createTransaction(Denomination.ARKTOSHI,  long: 1)`

would be the equivalent to:

 `createTransaction(Denomination.ARK, double: 0.00000001)`

**Pull request merged**